api/meetups/tests/test_meetups.py

```diff
@@ -169,20 +169,7 @@ def force_authenticate_user(self) -> None:
         """
         self.client.force_authenticate(user=self.user)
 
-    def test_successfull_creation_of_meetup(self) -> None:
```

Comment:
Hello @Bakley ,
would you return the test for successful creation of meetup like we agreed then you can just add the rest framework configuration in settings.py like we agreed.

Reply:
Yes, @Ogutu-Brian Done that, I am documenting the endpoint so will add with the changes in the next push